client/src/components/Home/AboutUs/Notice/Notice.css

```diff
@@ -39,17 +39,16 @@
     pointer-events: none;
 }
 
```

Comment:
Links in the notice are not noticeable, try giving it a link feel like some colors or underline or stuff like that!

Reply:
On it.

Reply:
Done.